src/content/docs/usage/prebuilds.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Prebuilds
+description: Learn how to manage and reuse Prebuilds across Workspaces.
+sidebar:
+  label: Prebuilds
+---
+
+Prebuilds are designed to speed up development by setting up a [project configuration](/usage/projects#project-configuration) that listens to changes in the connected Git Provider repository. This ensures a quicker project creation process when initiating a Workspace.
+
+Prebuilds work by registering a listener to webhook events from the Git Provider by sending a public API endpoint that the Git Provider will use to send requests.
+
+## Add a Prebuild
+
+1. Run the following command to add a new Prebuild:
+
+   ```shell
+   daytona prebuilds add
+   ```
+
+   Using the command opens a view that prompts you to select a project configuration and the Git branch you plan to work on. You can then decide on a commit interval after which a build should be triggered, as well as any specific trigger files whose changes should immediately start the build process.
```

Comment:
@stefanicjuraj @Tpuljak files? how do I define the files? just name? github path? what is the format, this is unclear to me

Reply:
> Enter full paths for files whose changes you want to explicitly trigger a prebuild. Use new lines for multiple entries.

This is written below. I wouldn't explain more here.